conbench/tests/api/test_compare.py

```diff
@@ -315,13 +315,38 @@ def test_compare_unknown_compare_ids(self, client):
         response = client.get("/api/compare/benchmarks/foo...bar/")
         self.assert_404_not_found(response)
 
+    @pytest.mark.parametrize(
+        ["baseline_result_id", "expected_z_score"],
+        [
+            # result on the fork point commit
+            (5, -2.1857497724635935),
+            # result on the parent commit
+            (6, -2.8246140882627757),
+            # result on the head commit of the default branch
+            (8, 0.12045310863100521),
+        ],
+    )
+    def test_compare_different_baselines(
+        self, client, baseline_result_id, expected_z_score
+    ):
+        self.authenticate(client)
+        _, benchmark_results = _fixtures.gen_fake_data()
+        contender = benchmark_results[7]
+        baseline = benchmark_results[baseline_result_id]
```

Comment:
I find interesting in this code, the baseline and contender are both "BenchmarkResult"s. For our choice of terminology: do we need to say "baseline result" and "contender result" instead, or is it established enough by now that we mean an individual benchmark result when we say e.g. "contennder"?